1-Authentication/2-sign-in-b2c/SPA/package.json

```diff
@@ -1,50 +1,42 @@
 {
-  "name": "ms-identity-angular-c1s2",
-  "version": "1.0.0",
-  "description": "Angular single-page application using MSAL React to sign-in users against Azure AD B2C",
+  "name": "spa",
+  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "test": "ng test --watch=false --no-progress --browsers=ChromeHeadlessCI",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
```

Comment:
Do we want this command back? There's some documentation for using this.

Reply:
Come again? 😃